### ISSUE TITLE
Auto-upgrade notifications.

### ISSUE THREAD

**Issue by sergiutesu** [umbraco/Umbraco.Cloud.Issues#577]
### Issue description

Feature request 📝
Currently, whenever there is a scheduled auto-upgrade there are no notifications in regards to the status of that auto-upgrade.

You can only see what is happening if you are logged into the Portal. 
On the project page at the time of the upgrade, where you should be able to see the upgrade status via the green bar at the bottom.

However, there is no email notification when an auto-upgrade starts, completes, or fails.

In certain situations, if there is an outgoing issue, the auto-upgrade will fail and users won't know about it. 
As a consequence, the project won't be upgraded.

Bottom line, it would be great to have some notifications related to scheduled auto-upgrades 🙌

**Comment by dawoe** [comment]
Hi @sergiutesu ,

The notification I am interested most at the moment is when the auto-upgrade fails. This could be sent to the technical contact of the project.

Dave

**Comment by sajumb** [comment]
Thanks for the feature request and comment. We will soon send out notifications to technical contacts when an auto-upgrade for a project fails.
